keyboards/anavi/knob1/keymaps/default/keymap.c

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 Leon Anavi <leon@anavi.org>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+#include <stdio.h>
+
+enum layer_names {
+    _BASE
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BASE] = LAYOUT_k1(
+        KC_MUTE
+    )
+};
+
+#ifdef OLED_ENABLE
+
+bool oled_task_user(void) {
```

Comment:
This should be moved the keyboard's c file, with:
```c
bool oled_task_kb(void) {
    if (!oled_task_user()) {
        return false;
    }
    // ...
```

Reply:
Thank you for the code review. Done. I've forced pushed the modified commit. Thanks, Leon